libs/ledger-live-common/src/manager/api.ts

```diff
@@ -168,7 +171,7 @@ const catalogForDevice: (params: {
 
     return data.map(mapApplicationV2ToApp);
   },
-  _ => "",
+  a => `${getEnv("MANAGER_API_BASE")}_${a.provider}_${a.targetId}_${a.firmwareVersion}`,
```

Comment:
We were always getting the same result before for `/v2/apps/by-target`, so the cache key was `""` but not anymore ?

Reply:
I think it was probably an issue